interpreter/variable/all.go

```diff
@@ -39,6 +39,17 @@ func NewAllScopeVariables(ctx *context.Context) *AllScopeVariables {
 	}
 }
 
+// return unescaped path value from the specified URL
+// this function makes the best effort to get the raw path
+// even when standard getEscapedPath() chooses escaped version
```

Comment:
Typo, `EscapedPath()`

Reply:
Thanks! Goes to show golang is not my primary language :)
Fixed.